plugins/digital_csic/plugin.py

```diff
@@ -13,80 +12,73 @@
 import api.utils as ut
 import sys
 
-logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, format='\'%(name)s:%(lineno)s\' | %(message)s')
-    
-logger = logging.getLogger(os.path.basename(__file__))
-
+logging.basicConfig(
+    stream=sys.stdout, level=logging.DEBUG, format="'%(name)s:%(lineno)s' | %(message)s"
+)
 
-class Plugin(Evaluator):
+logger = logging.getLogger(os.path.basename(__file__))
 
-    """
-    A class used to represent an Animal
 
-    ...
+class Digital_CSIC(Evaluator):
```

Comment:
Class need to be called Plugin(Evaluator)

Reply:
done